wandb/sdk/launch/launch.py

```diff
@@ -37,13 +38,15 @@ def resolve_agent_config(
         "base_url": api.settings("base_url"),
         "registry": {},
         "build": {},
-        "runner": {},
+        "runners": {"local-process": {}, "local-container": {}},
```

Comment:
@KyleGoyette ive decided if runners are omitted, the agent will support local process and container by default